Extensions/DiscordRichPresence.json

```diff
@@ -0,0 +1,372 @@
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The name of the small image",
+          "longDescription": "",
+          "name": "smallImageKey",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The text of the small image",
+          "longDescription": "",
+          "name": "smallImageText",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "An identifier of the player's party, lobby, or group\t",
+          "longDescription": "",
+          "name": "partyId",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The current size of the player's party, lobby, or group",
+          "longDescription": "",
+          "name": "partySize",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The maximum size of the player's party, lobby, or group\t",
+          "longDescription": "",
+          "name": "partyMax",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "A unique payload to give to the game when clicking the spectate button",
+          "longDescription": "",
+          "name": "spectateSecret",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "A unique payload to give to the game when clicking the join button",
+          "longDescription": "",
+          "name": "joinSecret",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Triggers once when a join request from another user was received.",
+      "fullName": "On join request",
+      "functionType": "Condition",
+      "name": "OnJoinRequest",
+      "private": false,
+      "sentence": "Join request received",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "eventsFunctionContext.returnValue = gdjs.evtTools.drp.joinRequest !== null;\r\n",
```

Comment:
Nitpicking: I know it's a bit of a pain, but even  on `evtTools`, `drp` is not a name that is unique enough. What if someone wants to make a "dynamic response plugin" in the future? Name your namespace "discordRichPresence" ;)